packages/spruce-types/src/gql/types/SprucebotAvatar.gql

```diff
@@ -0,0 +1,32 @@
+"Sprucebot's current state of mind"
+enum SprucebotAvatarStateOfMind {
```

Comment:
If this is answering the question "How does Sprucebot feel?" shouldn't the options be emotions? i.e. `calm`, `contemplative`, `curious`, `productive` respectively for the four options here.

Reply:
This feels like a nit, but it also seems kind of critical in getting the grokability of this on point